clex/driver.c

```diff
@@ -400,7 +400,13 @@ void reverse_toks (int idx)
   int matched = 0;
   int which = 0;
   int i;
+
+#ifdef _MSC_VER
+  int *saved = calloc(N, sizeof(int));
```

Comment:
ugh, does MSVC really not support C99-style arrays? I wonder if it would be better to use alloca() here so that there's no need to add calls to free?